tobaccocessation/templates/main/index.html

```diff
@@ -59,7 +59,7 @@
                     <h4>Course Materials</h4>
                     <ul>
                         <li>&nbsp;&nbsp;
-                            <a href="http://ccnmtl.columbia.edu/tobaccocessation/TobaccoCessation_CourseGuide_{{profile.role|capfirst}}.pdf" target="_blank" class="btn">Download Course Guide&nbsp;<i class="icon-download"></i>
+                            <a href="https://www1.columbia.edu/sec/ccnmtl/remote/static/tobaccocessation/TobaccoCessation_CourseGuide_{{profile.role|capfirst}}.pdf" target="_blank" class="btn">Download Course Guide&nbsp;<i class="icon-download"></i>
```

Comment:
Should these URLs be working?

I can't get to the old one:
* http://ccnmtl.columbia.edu/tobaccocessation/TobaccoCessation_CourseGuide_Student.pdf
or the new one:
* https://www1.columbia.edu/sec/ccnmtl/remote/static/tobaccocessation/TobaccoCessation_CourseGuide_Student.pdf